onvm/onvm_nflib/onvm_flow_table.h

```diff
@@ -148,7 +148,6 @@ onvm_ft_fill_key(struct onvm_ft_ipv4_5tuple *key, struct rte_mbuf *pkt) {
                 return -EPROTONOSUPPORT;
         }
         ipv4_hdr = onvm_pkt_ipv4_hdr(pkt);
-        memset(key, 0, sizeof(struct onvm_ft_ipv4_5tuple));
```

Comment:
When filling in the key, passing in a pointer within the pkt handler from a secondary process resulted in a seg fault.